airbyte-webapp/src/packages/cloud/views/layout/CloudMainView/CloudBottomItems.tsx

```diff
@@ -0,0 +1,52 @@
+import { FormattedMessage } from "react-intl";
+
+import { CreditsIcon } from "components/icons/CreditsIcon";
+
+import { FeatureItem, useFeature } from "hooks/services/Feature";
+import { CloudRoutes } from "packages/cloud/cloudRoutePaths";
+import { useGetCloudWorkspace } from "packages/cloud/services/workspaces/CloudWorkspacesService";
+import { LOW_BALANCE_CREDIT_THRESHOLD } from "packages/cloud/views/credits/CreditsPage/components/LowCreditBalanceHint/LowCreditBalanceHint";
+import { RoutePaths } from "pages/routePaths";
+import { useCurrentWorkspace } from "services/workspaces/WorkspacesService";
+import { MenuContent } from "views/layout/SideBar/components/MenuContent";
+import { NavItem } from "views/layout/SideBar/components/NavItem";
+import SettingsIcon from "views/layout/SideBar/components/SettingsIcon";
+
+import { CloudResourcesDropdown } from "./CloudResourcesDropdown";
+import { CloudSupportDropdown } from "./CloudSupportDropdown";
+
+export const CloudBottomItems: React.FC = () => {
+  const workspace = useCurrentWorkspace();
+  const cloudWorkspace = useGetCloudWorkspace(workspace.workspaceId);
+  const isAllowUpdateConnectorsEnabled = useFeature(FeatureItem.AllowUpdateConnectors);
+
+  return (
+    <ul>
```

Comment:
Noticed while reading Edmundo's comment - `<div>` is actually not a valid child element of a `<ul>`. It needs to contain only `<li>`. Could we make `<MenuContent>` a `<ul>` to avoid this? It might mean we can't use the `<FlexContainer>` 🤷

Reply:
Oh!  Good catch!
1. Yes with a tad of reworking
2. This kind of thing seems (valid html children) like it should be lintable but I couldn't find anything on it

Reply:
3. as long as I'm tackling that, it makes sense to add the children.map like Edmundo outlined (since I'm having to switch up the one place I was using this as a div anyways)

Reply:
addressed in https://github.com/airbytehq/airbyte/pull/21227/commits/23f818a2bb790415023f030d343e9bbe14d11546

Reply:
addressed in https://github.com/airbytehq/airbyte/pull/21227/commits/23f818a2bb790415023f030d343e9bbe14d11546